lib/chef/provisioning/convergence_strategy/precreate_chef_objects.rb

```diff
@@ -2,11 +2,20 @@
 require 'pathname'
 require 'cheffish'
 require 'chef_zero/socketless_server_map'
+require_relative 'ignore_convergence_failure'
 
 class Chef
 module Provisioning
   class ConvergenceStrategy
     class PrecreateChefObjects < ConvergenceStrategy
+
+      def initialize(convergence_options, config)
+        super
+        if convergence_options[:ignore_failure]
```

Comment:
`ignore_failure` can now accept a Fixnum, a Range, a custom error, or some array with a combination of those.  Or it can accept `true` to ignore all RuntimeErrors.  The documentation has been updated to show this.